uvicorn/protocols/http/h11_impl.py

```diff
@@ -96,6 +109,11 @@ def connection_made(  # type: ignore[override]
         self.client = get_remote_addr(transport)
         self.scheme = "https" if is_ssl(transport) else "http"
 
+        if self.config.is_ssl:
+            self.tls = get_tls_info(transport)
```

Comment:
Given that `uvicorn` requires Python >= 3.8, I'd use the Walrus operator here to increase readability.